.gitignore

```diff
@@ -2,4 +2,10 @@ lib/
 node_modules/
 bundle.css
 bundle.js
-.idea
\ No newline at end of file
+.idea
+yarn.lock
+.vscode/launch.json
+.vscode/settings.json
+.env
+.prettierrc
```

Comment:
this one should not be ignored. If you use some settings for prettier, then commit it. Or use default settings for prettier

Reply:
I like no-semi, my .prettierrc has no semi. What should I do, commit?